plugins/azure_sentinel/icon_azure_sentinel/util/api.py

```diff
@@ -259,3 +266,396 @@ def list_entities(
         )
         results = self._list_all("POST", final_uri, type_="entities")
         return [return_non_empty(entity) for entity in results]
+
+    def get_comment(
+        self,
+        incident_id: str,
+        incident_comment_id: str,
+        resource_group_name: str,
+        workspace_name: str,
+        subscription_id: str,
+        api_version: str = "2021-04-01",
+    ):
+        uri = Endpoint.GETCOMMENT
+        final_uri = uri.format(
+            subscription_id,
+            resource_group_name,
+            workspace_name,
+            incident_id,
+            incident_comment_id,
+            api_version,
+        )
+        _, result = self._call_api("GET", final_uri, self.headers)
+        return return_non_empty(result)
+
+    def list_comments(
+        self,
+        incident_id: str,
+        resource_group_name: str,
+        workspace_name: str,
+        subscription_id: str,
+        api_version: str = "2021-04-01",
```

Comment:
Is this string a safe default value? I see in other places the API version is `2021-10-01`. Shouldn't these be the same throughout this file?